### ISSUE TITLE
Battle mode

### ISSUE THREAD

**Issue by laggron42** [laggron42/BallsDex-DiscordBot#1]
Make use of the stats and capacities for a battle mode.

**Comment by TheBigDipster** [comment]
@laggron42 I'd like to work on this with a friend.

**Comment by laggron42** [comment]
@TheBigDipster Hey, sorry I forgot to check my pings, are you still interested? You can contact me on Discord if that's the case, join https://discord.gg/ballsdex and ask a moderator to put you in contact with me.

Also wanted to add, Discord is opening the SDK for voice chat games, and I think this is the perfect way to implement the interface.

**Comment by ezykill10** [comment]
will this ever get added...

**Comment by imtherealF1** [comment]
> will this ever get added...

yes @ezykill10, but theres a lot of planning needed, so its going to take time, be patient
